### ISSUE TITLE
What happened to the sidebar in the torch.nn documentation?

### ISSUE THREAD

**Issue by mhdadk** [pytorch/pytorch#39858]
## 📚 Documentation

![Annotation 2020-06-11 095959](https://user-images.githubusercontent.com/17471410/84400584-38a36100-abd0-11ea-8791-e507deca05be.png)

There used to be a sidebar in the torch.nn page with headings such as "Convolutional layers" and "Loss functions" that could be expanded to jump quickly to these functions. This means that there were 3 ways of navigating the page:

1. Clicking on these headings and links.
2. Scrolling manually.
3. Searching for function names such as "nn.Conv2d" using ctrl+F.

Now, however, with the sidebar gone, there are only two ways, which are extremely inconvenient and inefficient:

1. Scrolling manually.
2. Searching for function names such as "nn.Conv2d" using ctrl+F.

Why was the sidebar removed and can it be brought back?

**Comment by aadhav-n1** [comment]
It does seem to work for me. Screenshot took at 11:59:37 EDT.

URL: https://pytorch.org/docs/stable/nn.html

![torch.nn](https://user-images.githubusercontent.com/53528139/84409175-b3996580-ac2a-11ea-8a24-5f69a2545cce.png)

**Comment by ShawnZhong** [comment]
Assuming you are using a desktop browser, you can make your window wider to see the sidebar. I think it’s hidden for mobile devices for space constraints.

**Comment by mhdadk** [comment]
@ShawnZhong I just needed to zoom out! Thanks a lot that solved my problem.
